### ISSUE TITLE
Clone and go a no go

### ISSUE THREAD

**Issue by jskulavik** [isaacabraham/ServiceFabricFsDemo#1]
Hi @isaacabraham: I'm attempting to run down a clone and go issue with this repo.  After cloning and running paket update, I get various build errors, the most significant of which is 'StatefulActor' is not defined.  I'm using VS2015 Update 2.  I'll continue to try and fix my build, but wanted you to know that I may get lucky and submit a pull request to update the VS solution to the latest and greatest so that others can clone and go.

**Comment by isaacabraham** [comment]
Yes please do. I suspect that this repo is out of date with the latest SDK anyway - perhaps a Paket update and then fix the types will be required.

**Comment by jskulavik** [comment]
Just a quick update @isaacabraham:  It appears that the latest SF SDK has removed StatefulActor from the Microsoft.ServiceFabric.Actors assembly.  I'm going to try and find it's replacement or new home and use this as an exercise to teach myself both more SF and your F# "philosophy" around SF highlighted in your blog posts and this repo.  I may need to hit you up for a bit of guidance along the way...

**Comment by isaacabraham** [comment]
Yes, I suspected as much. Having had a quick look at the new SDK it does look like the changes are pretty sweeping - I couldn't find a generic `Actor<T>` or anything, so it looks like a bit of work to get things working.

**Comment by jskulavik** [comment]
Yep, Actor has been "bumped" down into Microsoft.ServiceFabric.Actors.Runtime and we need to hunt down the state management methods and "convert" them to functions no?

**Comment by isaacabraham** [comment]
Basically yes. There's no longer a State property - instead there appears to be a StateManager on which you can call methods like Get and TryGet etc.

**Comment by jskulavik** [comment]
Okay, thanks.  That's what I understood.  Off we go... :-)
